.github/workflows/tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.6", "3.7", "3.8", "3.9"]
```

Comment:
https://github.com/encode/django-rest-framework/blob/master/tox.ini#L6 looks like need to exclude python 3.6 3.7 for django4
yep, also can add 3.10 and exclude that for "2.2", "3.0", "3.1"
also I see DRF stopped testing for 3.0 version, can remove everywhere 3.0 too